packages/file-format/schema/user.schema.yaml

```diff
@@ -13,6 +13,8 @@ properties:
       - expandedTextStylePathsInPopover
       - libraryListCollapsed
     properties:
+      componentSidebarTreeStructure:
+        { $ref: ./enums/component-sidebar-tree-structure.yaml }
```

Comment:
You need to name this file ending `.schema.yaml` for it to be picked up properly by the built scripts.